### DISCUSSION TITLE
Is there any documentation/example about how to integrate strimzi with datadog?

### DISCUSSION THREAD

**Discussion by jinzishuai** [strimzi/strimzi-kafka-operator#4971] [General]
I did some search and found out that some discussion such as https://github.com/strimzi/strimzi-kafka-operator/discussions/4287 mentioned that the broker/connect should be able to work with datadog via JMX even though this still a problem with zookeeper.
In fact, I can probably live without having datadog monitoring for zookeeper but monitoring on the broker/connect/connectors/topics are very important to us.

I wonder if there is more information that I could use as a guidance to set up my k8s environment with datadog and strimzi.  Thank you very much.

**Comment by scholzj** [comment]
Can't DataDog consume Prometheus / OpenMetrics metrics? I think that was something I discussed in the past with some other DataDog user. Otherwise the state described in #4287 is still valid. JMX is supported for Kafka and Connect (and I guess for Kafka Mirror Maker 2 since that is based on Connect). There is no support for Zookeeper.

**Reply by jinzishuai** [reply]
thank you. I'll look into the Prometheus monitoring first.
